Packs/FeedAzureADConnectHealth/Integrations/AzureADConnectHealthFeed/AzureADConnectHealthFeed.py

```diff
@@ -0,0 +1,198 @@
+    Returns:
+        Indicators.
+    """
+    iterator = client.build_iterator()
+    indicators = []
+    if limit > 0:
+        iterator = iterator[:limit]
+    for item in iterator:
+        value = item.get('value')
+        type_ = item.get('type', FeedIndicatorType.Domain)
+        raw_data = {
+            'value': value,
+            'type': type_,
+        }
+        for key, val in item.items():
+            raw_data.update({key: val})
+        indicator_obj = {
+            'value': value,
+            'type': type_,
+            'service': 'Azure AD Connect Health Feed',
+            'rawJSON': raw_data,
+        }
+        if feed_tags:
+            indicator_obj['fields'] = {
+                'tags': feed_tags
+            }
+        indicators.append(indicator_obj)
+    return indicators
+
+
+def get_indicators_command(client: Client,
+                           params: Dict[str, str],
+                           args: Dict[str, str]
+                           ) -> Tuple[str, Dict[Any, Any], Dict[Any, Any]]:
+    """Wrapper for retrieving indicators from the feed to the war-room.
+    Args:
+        client: Client object with request
+        params: demisto.params()
+        args: demisto.args()
+    Returns:
+        Outputs.
+    """
+    feed_tags = argToList(params.get('feedTags', ''))
+    limit = int(args.get('limit', '10'))
+    indicators = fetch_indicators(client, feed_tags, limit)
+    human_readable = tableToMarkdown('Indicators from Microsoft Azure Feed:', indicators,
+                                     headers=['value', 'type'], removeNull=True)
+
+    return human_readable, {}, {'raw_response': indicators}
+
+
+def fetch_indicators_command(client: Client, params: Dict[str, str]) -> List[Dict]:
+    """Wrapper for fetching indicators from the feed to the Indicators tab.
+    Args:
+        client: Client object with request
+        params: demisto.params()
+    Returns:
+        Indicators.
+    """
+    feed_tags = argToList(params.get('feedTags', ''))
+    indicators = fetch_indicators(client, feed_tags)
+    return indicators
+
+
+def main():
+    """
+    PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    params = demisto.params()
+    base_url = params.get('url')
+    insecure = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+
+    command = demisto.command()
+    demisto.info(f'Command being called is {command}')
+
+    try:
+        client = Client(
+            base_url=base_url,
+            verify=insecure,
+            proxy=proxy,
+        )
+
+        commands: Dict[
+            str, Callable[[Client, Dict[str, str], Dict[str, str]], Tuple[str, Dict[Any, Any], Dict[Any, Any]]]
+        ] = {
+            'test-module': test_module,
+            'azure-ad-health-get-indicators': get_indicators_command
+        }
+        if command in commands:
+            return_outputs(*commands[command](client, demisto.params(), demisto.args()))
```

Comment:
we are now using `return_results` and `CommandResults` class.
https://xsoar.pan.dev/docs/integrations/context-and-outputs
Not a must, but to your consideration.

Reply:
I'll look into it for next contribution